From c26811a07a272622b41afc1591f3e96f991191e7 Mon Sep 17 00:00:00 2001
From: islemk69 <islemkaismoun@gmail.com>
Date: Mon, 2 Mar 2026 16:31:50 +0100
Subject: [PATCH] kbuild: add -thor_kernel to EXTRAVERSION

Signed-off-by: islemk69 <islemkaismoun@gmail.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 244608598..2251b1af7 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 7
 PATCHLEVEL = 0
 SUBLEVEL = 0
-EXTRAVERSION = -rc2
+EXTRAVERSION = -thor_kernel
 NAME = Baby Opossum Posse
 
 # *DOCUMENTATION*
-- 
2.52.0

